src/api/get-installations.js

```diff
@@ -8,6 +8,24 @@ import ALIASES from '../aliases';
 // Installation info cache
 var installationsCache = null;
 
+// Original code page value
+var origCodePage = null;
```

Comment:
Too complicated. It can be done in much simpler manner. Here is a tip: if we can set desired code page and obtain results in single `cmd` execution maybe we can restore original code page there as well?